cli/running/cluster_instance.go

```diff
@@ -124,6 +127,10 @@ func (inst *clusterInstance) Start() error {
 			return err
 		}
 		cmd.Env = append(cmd.Env, "TT_CONSOLE_SOCKET_DEFAULT="+consoleSocket)
+		_, listenSet := os.LookupEnv("TT_LISTEN")
```

Comment:
We need to check that TT_LISTEN is not set, so we don't  overwrite it if user set it.

Reply:
Because we don't want to overwrite it if user set it.
e.g
```
better0fdead@better0fdead:~/tarantool-ex$ cat exd_simple_working/config.yaml 
credentials:
  users:
    guest:
      roles: [super]

log:
  to: file
  file: tnt_{{ instance_name }}.log

groups:
  group-001:
    replicasets:
      replicaset-001:
        instances:
          master:
            database:
              mode: rw
            wal:
              dir: ./var/lib/{{ instance_name }}_wal
            snapshot:
              dir: ./var/lib/{{ instance_name }}_snapshot
            console:
              socket: ./run_{{ instance_name }}.control
            process:
              pid_file: ./run_{{ instance_name }}.pid
            app:
              file: 'master.lua'
better0fdead@better0fdead:~/tarantool-ex$ cat exd_simple_working/master.lua 
box.schema.create_space('users', {
    format = {
        {name = 'id', type = 'unsigned'},
        {name = 'first name', type = 'string'},
        {name = 'second name', type = 'string', is_nullable = true},
        {name = 'age', type = 'number', is_nullable = false},
    }
})

box.space.users:create_index('primary', {
    parts = {
        {field = 1, type = 'unsigned'},
    },
})

box.space.users:insert{2, 'Fay', 'Rivers', 41}
box.space.users:insert{3, 'Zachariah', nil, 13}
box.space.users:insert{4, 'Milo', 'Walters', 74}

box.schema.func.create('forever_true_test', {
    body = "function(...) return true end",
    if_not_exists = true
})

better0fdead@better0fdead:~/tarantool-ex$ cat exd_simple_working/instances.yml 
master:
better0fdead@better0fdead:~/tarantool-ex$ pwd
/home/better0fdead/tarantool-ex
better0fdead@better0fdead:~/tarantool-ex$ export TT_LISTEN=/home/better0fdead/tarantool-ex/sock.lua
better0fdead@better0fdead:~/tarantool-ex$ tarantool --config exd_simple_working/
config.yaml    instances.yml  master.lua     override/      
better0fdead@better0fdead:~/tarantool-ex$ tarantool --config exd_simple_working/config.yaml --name master
started
2024-02-22 03:47:57.669 [8740] main F> fatal error, exiting the event loop
better0fdead@better0fdead:~/tarantool-ex$ ls -al
total 24
drwxrwxr-x  4 better0fdead better0fdead 4096 фев 22 03:47 .
drwxr-xr-x 36 better0fdead better0fdead 4096 фев 22 03:45 ..
drwxrwxr-x  3 better0fdead better0fdead 4096 фев 22 03:46 exd_simple_working
srwxrwxr-x  1 better0fdead better0fdead    0 фев 22 03:47 run_master.control
-rw-r--r--  1 better0fdead better0fdead    4 фев 22 03:47 run_master.pid
srwxrwxr-x  1 better0fdead better0fdead    0 фев 22 03:47 sock.lua
-rw-r-----  1 better0fdead better0fdead 3180 фев 22 03:47 tnt_master.log
drwxrwxr-x  3 better0fdead better0fdead 4096 фев 22 03:47 var
```

Reply:
It should not be overwritten by _DEFAULT vars.

Reply:
changed to IPROTO_LISTEN_DEFAULT